lib_mic_array/lib_build_info.cmake

```diff
@@ -0,0 +1,15 @@
+set(LIB_NAME lib_mic_array)
+set(LIB_VERSION 5.2.0)
```

Comment:
If this version value is for lib_mic_array, it needs to be `5.3.0`.

Reply:
@mbanth you're absolutely right, have opened https://github.com/xmos/lib_mic_array/pull/221 to fix on develop and will then merge that into this branch

Reply:
@mbanth okay, that should be on this branch now

Reply:
Agreed, the changes resolve this issue.